### PR TITLE
Updated code to published library in use

### PR DESCRIPTION
**Pull request by notmariazoe**
Sorry for the XXXL PR, includes all the changes we did for the first release to partners on sust topics.
Will likely have to add a few extra adjustments for the linters to let me merge.

### CODE REVIEW

**Review by adamread — COMMENTED** [review]
I think most of the files are missing the license header. That should be added before this is merged. I'm fine with approving once that's done.

I'm also not a fan of using `newDeckId` as a function parameter in most places. For someone new skimming the code, it seems like you're creating new decks all the time.

File: src/sls_core.js
Comment (notmariazoe) [reply src/sls_core.js:313]: Sure, we can add that as a check to see whether someone might have added a base64 encoded png. For now, it's only checking for jpeg as it's the case where the LH response from PSI API would return that. Might as well encapsulate this somewhere else. I'll add it as an issue so we can tackle in the future, thanks for flagging Adam.

File: src/sls_facades.js
Comment (notmariazoe) [reply src/sls_facades.js:50]: There's two things happening here, the error is being logged twice due to a merging issue (yikes!), and the logging format comes straight from the Sheets docs, but I found it to not be very informative regardless when using the debugger in the AppsScript editor. Will remove the duplicate line and add an issue to look into this, thank you for flagging.

Comment (notmariazoe) [reply src/sls_facades.js:50]: You might have seen this on other parts of the code, where this was being used before extracting it to the facades -library?- file so that any audit type could use it. A lot of these audits aren't being used anymore, so I figure we could always deprecate them and delete them.

File: src/sls_sustainability.js
Comment (notmariazoe) [reply src/sls_sustainability.js:156]: As discussed elsewhere, this is a framework function that allows every specific audit type (or implementation of katalyst, it doesn't have to be an audit) to apply whatever modifications they want at the end of the deck creation. The naming isn't great either, as it should be something like "postDeckCreationProcessing" or the likes.

Will add an issue to document and phrase better on a separate PR.

File: src/sls_sustainability.js
Comment (notmariazoe) [reply src/sls_sustainability.js:71]: Done

File: src/sls_psi_api_connector.js
Comment (notmariazoe) [reply src/sls_psi_api_connector.js:24]: Good point, thanks.

File: src/sls_psi_api_connector.js
Comment (notmariazoe) [reply src/sls_psi_api_connector.js:240]: This was commented out because it wasn't used. This version of the PSI API connector deviates from Xavier's version. Will delete the commented out bit.

File: generate.sh
Comment (notmariazoe) [reply generate.sh:9]: You're right. This is also due some naming changes (we discussed removing the sls prefix), so will add it as issue and review then.

File: src/sls_core.js
Comment (adamread) [inline src/sls_core.js:1]: I think this file is still missing the license header.

File: src/sls_sustainability.js
Comment (adamread) [inline src/sls_sustainability.js:1]: Still missing the license header.

**Review by adamread — COMMENTED** [review]
Two files still need a license header. Since the approval gets taken away when you make changes, I'll wait for the next notification.

**Review by adamread — APPROVED** [review]
The test file also needs a license, but I'm sure you'll get to it =)

**Review by adamread — APPROVED** [review]
LGTM

**Pull request merged**
